gradle.properties

```diff
@@ -16,6 +16,6 @@ POM_URL=https://bugsnag.com
 POM_ARTIFACT_ID=bugsnag-android
 POM_PACKAGING=jar
 
-ANDROID_BUILD_TARGET_SDK_VERSION=19
-ANDROID_BUILD_TOOLS_VERSION=21.1.1
-ANDROID_BUILD_SDK_VERSION=19
```

Comment:
Just because im curious: Why are these 3 values defined here and not in the build.gradle?

/cc @loopj